pydy/codegen/tests/test_c_code.py

```diff
@@ -105,7 +105,17 @@ def test_generate_comma_lists(self):
                     'v0(t), v1(t), v2(t), v3(t), v4(t), v5(t)',
                     'f5(t), f2(t), f3(t), f0(t), f4(t), f1(t)')
 
-        assert self.generator.comma_lists() == expected
+        actual = self.generator.comma_lists()
```

Comment:
Not for the last term. Originally I assumed the order would not be always known, but as the fourth arguments is now a list of the ordered terms, we could.